book.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './rental'
+
+class Book
+  def initalize(title, author)
+    @title = title
+    @author = author
+  end
+
+  attr_accessor :title, :author
+
+  has_many :book
```

Comment:
- [ ] I see you're using active record associations in a plain ruby app. Please note that this will throw an error especially since there's not a Gemfile that includes the ActiveRecord gem.
If you want to use ActiveRecord in a regular Ruby app. Please follow this [guide](https://www.techcareerbooster.com/blog/use-activerecord-in-your-ruby-project#:~:text=ActiveRecord%20is%20a%20gem%20that,RDBMS%2C%20like%20MySQL%20or%20PostgreSQL.).